test/mocks/event/mocks.cc

```diff
@@ -10,12 +10,14 @@ using testing::NiceMock;
 using testing::Return;
 using testing::ReturnNew;
 using testing::ReturnPointee;
+using testing::ReturnRef;
 using testing::SaveArg;
 
 namespace Envoy {
 namespace Event {
 
 MockDispatcher::MockDispatcher() {
+  ON_CALL(*this, timeSource()).WillByDefault(ReturnRef(time_system_));
```

Comment:
I've added the ability to use a different time source here- if there is a preferred way to do this please let me know.